example-adapter/dependency-check-suppressions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When running the build, I also got the following message:

```
One or more dependencies were identified with known vulnerabilities in Example Adapter for X-Road version 6:

jackson-core-2.13.5.jar (pkg:maven/com.fasterxml.jackson.core/jackson-core@2.13.5, cpe:2.3:a:fasterxml:jackson-modules-java8:2.13.5:*:*:*:*:*:*:*, cpe:2.3:a:json-java_project:json-java:2.13.5:*:*:*:*:*:*:*) : CVE-2022-45688
```

Should that be added here as well?

Reply:
I updated the `dependency-check-maven` plugin to version 8.1.2 and this vulnerability disappeared.